### DISCUSSION TITLE
648 aesthetic modifiers study, based on Aesthetics Wiki

### DISCUSSION THREAD

**Discussion by Jonseed** [adieyal/sd-dynamic-prompts#120] [Show and tell]
I did a study of the 648 aesthetic modifiers in general_aesthetics.txt in the Jumbo collection using combinatorial generation. Some might find it interesting:

[https://www.reddit.com/r/StableDiffusion/comments/zon9zn/648_aesthetic_modifiers_study_based_on_aesthetics/](https://www.reddit.com/r/StableDiffusion/comments/zon9zn/648_aesthetic_modifiers_study_based_on_aesthetics/)

**Comment by adieyal** [comment]
Very cool - if you have the energy, you should put it up on medium which might reach more audiences. I'm going to put a link to your post in the aesthetics file.
